index.html

```diff
@@ -3,7 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>"Hello Microverse" project</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>"Hello Microverse!"</h1>
+    </header>
```

Comment:
Your header doesn't have any color, seem like you forgot to link your `styles.css` file with your `index.html`. 😉